### DISCUSSION TITLE
Easier & more comprehensive annotations

### DISCUSSION THREAD

**Discussion by rccoleman** [blakeblackshear/frigate#6104] [Ideas]
As I'm annotating images on Frigate+, there are a few usability features that I'd like to suggest:

- On the page that shows all the images, indicate which have been annotated and which have not.  It doesn't look like there's any way to know which I've already completed and which I still need to do without opening each individually.
- When annotating images, I wish that it would let me auto-advance to the next unlabeled image so that I can quickly and easily move through all new images
- If the plan is to add more labels in the future, I suggest adding them *now* so that we do one pass through images, rather than doing it again and again as new labels are added.

The easier it is to annotate images, the more likely you'll get useful data.

**Comment by blakeblackshear** [comment]
Thanks. All of these enhancements are on my list already. I appreciate the feedback.

**Comment by georgrin** [comment]
So good propostions :)
